CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## v0.6.1 (September 19. 2023)
+Features:
```

Comment:
we will be also releasing support for BAH cluster resource and data-source, i.e csp subscription ID and service account id

Reply:
oh yes thanks